src/main/java/org/jenkinsci/lib/envinject/service/EnvInjectActionRetriever.java

```diff
@@ -23,15 +25,15 @@
     //Returns the abstract class Action due to a class loading issue
     //with EnvInjectAction subclasses. Subclasses cannot be casted from
     //all point of Jenkins (classes are not loaded in some points)
-    public Action getEnvInjectAction(AbstractBuild<?, ?> build) {
+    public Action getEnvInjectAction(@Nonnull AbstractBuild<?, ?> build) {
         List<Action> actions;
         if (build == null) {
```

Comment:
If build is non-nullable now this line can be removed.

Reply:
It's a legacy API issue, some plugins were trying to call it. That's why I keep NPE explicit